server/http.go

```diff
@@ -289,16 +293,15 @@ func (p *Plugin) postMeeting(creator *model.User, meetingID int, channelID strin
 }
 
 func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
-
 	userID := r.Header.Get("Mattermost-User-Id")
 	if userID == "" {
 		http.Error(w, "Not authorized", http.StatusUnauthorized)
 		return
 	}
 
 	var req startMeetingRequest
-	err := json.NewDecoder(r.Body).Decode(&req)
-	if err != nil {
+	var err error
+	if err = json.NewDecoder(r.Body).Decode(&req); err != nil {
```

Comment:
Nit:
```suggestion
	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
```

Reply:
This was needed, and in absence of it, linter was throwing an error.